Instructions/Labs/13-implement-transitions-in-tsql.md

```diff
@@ -241,6 +241,62 @@ END CATCH
 
 8. Switch to the query window that contains code to select the most recently modified customer and run it to verify that a record for *Ann Othercustomer* has been inserted. The transaction succeeded and was not rolled back, even though an error subsequently occurred.
 
+## Explore transaction concurrency
+
+On-premises SQL Server has a default isolation level of **READ_COMMITTED_SNAPSHOT_OFF**. This level of isolation will hold locks on rows while a transaction is acting on it. For example, inserting a customer into the customer table. If the update takes a long time to run, any queries on that table will be blocked from running until the transaction is committed or rolled back.

```

Comment:
We're introducing on-premises instances - is the isolation level different in others?

Reply:
Yes by default SQL Server on Azure is set to on.